src/Java.Interop.Tools.Generator/Java.Interop.Tools.Generator.csproj

```diff
@@ -13,10 +12,6 @@
     <OutputPath>$(UtilityOutputFullPath)</OutputPath>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Compile Include="..\utils\NullableAttributes.cs" />
```

Comment:
Since we're removing use of `NullableAttributes.cs`, shouldn't we also delete `src/utils/NullableAttributes.cs`?

Reply:
There are a few projects that must remain on `netstandard2.0` because they are consumed by `Xamarin.Android.Build.Tasks`, which must remain `netstandard2.0` so it can be loaded by VS MSBuild.  These projects still need `NullableAttributes.cs`.

For example:
- `Java.Interop.Localization.csproj`
- `Java.Interop.Tools.Diagnostics.csproj`
- `Java.Interop.Tools.JavaCallableWrappers.csproj`